### PR TITLE
remove kelsey's EC2 instances

### PR DESCRIPTION
**Pull request by zaro0508**

### CODE REVIEW

**Review by ahayden — APPROVED** [review]
Approved pending review from @kelshmo

**Review by kelshmo — APPROVED** [review]
There is an additional windows instance not listed here.

**Comment by zaro0508** [comment]
thanks, for the heads up @kelshmo,  the `kelsey-windows` instance will be remove as part of this PR.

**Pull request merged**
